Workbooks/UserEntityBehaviorAnalytics.json

```diff
@@ -0,0 +1,772 @@
+      "name": "text - 13"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json":
+          "### Persistence\r\n\r\nThe adversary is trying to maintain their foothold.\r\n\r\nPersistence consists of techniques that adversaries use to keep access to systems across restarts, changed credentials, and other interruptions that could cut off their access. Techniques used for persistence include any access, action, or configuration changes that let them maintain their foothold on systems, such as replacing or hijacking legitimate code or adding startup code.\r\n\r\n[Learn More Here](https://attack.mitre.org/tactics/TA0003/)"
+      },
+      "conditionalVisibility": {
+        "parameterName": "TacticM",
+        "comparison": "isEqualTo",
+        "value": "Persistence"
+      },
+      "customWidth": "30",
+      "name": "text - 13 - Copy"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json":
+          "### Discovery\r\n\r\nThe adversary is trying to figure out your environment.\r\n\r\nDiscovery consists of techniques an adversary may use to gain knowledge about the system and internal network. These techniques help adversaries observe the environment and orient themselves before deciding how to act. They also allow adversaries to explore what they can control and what’s around their entry point in order to discover how it could benefit their current objective. Native operating system tools are often used toward this post-compromise information-gathering objective.\r\n\r\n[Learn More Here](https://attack.mitre.org/tactics/TA0007/)"
+      },
+      "conditionalVisibility": {
+        "parameterName": "TacticM",
+        "comparison": "isEqualTo",
+        "value": "Discovery"
+      },
+      "customWidth": "30",
+      "name": "text - 13 - Copy - Copy - Copy - Copy"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json":
+          "### Collection\r\n\r\nThe adversary is trying to gather data of interest to their goal.\r\n\r\nCollection consists of techniques adversaries may use to gather information and the sources information is collected from that are relevant to following through on the adversary's objectives. Frequently, the next goal after collecting data is to steal (exfiltrate) the data. Common target sources include various drive types, browsers, audio, video, and email. Common collection methods include capturing screenshots and keyboard input.\r\n\r\n[Learn More Here](https://attack.mitre.org/tactics/TA0009/)"
+      },
+      "conditionalVisibility": {
+        "parameterName": "TacticM",
+        "comparison": "isEqualTo",
+        "value": "Collection"
+      },
+      "customWidth": "30",
+      "name": "text - 13 - Copy - Copy"
+    }
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/7b76bfbc-cb1e-4df1-b6e8-b826eef6c592/resourcegroups/soc/providers/microsoft.operationalinsights/workspaces/cybersecuritysoc"
+  ],
+  "fromTemplateId": "sentinel-UserWorkbook",
```

Comment:
change this parameter to "fromTemplateId": "Sentinel-UserEntityBehaviorAnalytics"